command/genesis/polybft_params.go

```diff
@@ -156,13 +161,17 @@ func (p *genesisParams) generatePolyBftChainConfig(o command.OutputFormatter) er
 		}
 	}
 
-	for _, burnContract := range p.burnContracts {
-		block, addr, err := parseBurnContractInfo(burnContract)
-		if err != nil {
-			return err
-		}
+	if len(p.burnContracts) > 0 {
```

Comment:
Do you need to set BaseFee and BaseFeeEM here as well?

```go
chainConfig.Genesis.BaseFee = command.DefaultGenesisBaseFee
chainConfig.Genesis.BaseFeeEM = command.DefaultGenesisBaseFeeEM
```

Reply:
Good point, done